cloud/services/virtualmachines/virtualmachines_test.go

```diff
@@ -41,7 +41,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
-const expectedInvalidSpec = "invalid vm specification"
+const expectedInvalidSpec = "invalid VM specification"
 
```

Comment:
There is another test in this file: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/cloud/services/virtualmachines/virtualmachines_test.go#L170 that also needs the expected error to change from `vm my-vm not found: #: Not found: StatusCode=404` to `VM my-vm not found: #: Not found: StatusCode=404`